openthread/src/lib.rs

```diff
@@ -1209,7 +1221,7 @@ impl<'a> OtContext<'a> {
     fn state(&mut self) -> &mut OtActiveState<'a> {
         unsafe {
             core::mem::transmute::<&'static mut OtActiveState<'static>, &mut OtActiveState<'a>>(
-                OT_ACTIVE_STATE.0.get().as_mut().unwrap().as_mut().unwrap(),
+                unwrap!(unwrap!(OT_ACTIVE_STATE.0.get().as_mut()).as_mut()),
```

Comment:
Wow

Reply:
The price to pay for doing unsafe stuff.
I might actually put some human-readable text for when the unwrap fails.